storage/innobase/dict/dict0dict.cc

```diff
@@ -914,6 +914,43 @@ dict_table_autoinc_unlock(
 {
 	mutex_exit(table->autoinc_mutex);
 }
+
+/********************************************************************//**
+Create and initialize the analyze index lock for a given table.
```

Comment:
Use doxygen style
```
/** Create
```

Reply:
You can mention that the lock is used to serialize two concurrent analyze index operations.

Reply:
done